### PR TITLE
Release/v0.22.2

### PR DESCRIPTION
**Pull request by carpawell**
Update `neofs-node` to `v0.22.2`. Add generated TLS certs to gitignore.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,3 +4,5 @@ tmp
 .secrets
 sites/*
 !sites/.gitkeep
+services/storage/*.crt
+services/storage/*.key
```
Comment (alexvanin) [inline .gitignore:8]: Why `*.key`? Cert is generated in startup, but keys are not.

Comment (alexvanin) [reply .gitignore:8]: My bad, they are. However we already use .key files for storage nodes. I suggest you to ignore keys with tls postfix.

Comment (carpawell) [reply .gitignore:8]: on `make up` i had two files generated locally: `s04tls.crt` and `s04tls.key`

i guess it is generated with `-keyout` flag [here](https://github.com/nspcc-dev/neofs-dev-env/blob/master/services/storage/artifacts.mk#L16) and used [here](https://github.com/nspcc-dev/neofs-dev-env/blob/master/services/storage/docker-compose.yml#L121)

Comment (carpawell) [reply .gitignore:8]: oh, comments were not updated

so you mean change it to `*tls.key`?

Comment (alexvanin) [reply .gitignore:8]: > so you mean change it to *tls.key?

Yes, it will not affect storage node keys this way.

Comment (carpawell) [reply .gitignore]: in fact, i hastened to exclude all keys and did not notice that there were more keys in that dir, thank you

changed to `*tls.*` both for cert and key

Comment (carpawell) [reply .gitignore]: in fact, i hastened to exclude all keys and did not notice that there were more keys in that dir, thank you

changed to `*tls.*` both for cert and key

**Review by alexvanin — APPROVED** [review]

**Pull request merged**